kb/migrating-visibility-data-from-es6.md

```diff
@@ -0,0 +1,82 @@
+---
+slug: migrate-visibility-data-from-es6
+title: Migrate visibility data from ES6
+tags:
+  - kb-article
+date: 2022-29-08T00:00:00Z
+---
+
+We added support for [Elasticsearch v7+ (ES7)](https://www.elastic.co/downloads/past-releases/#elasticsearch) in the v1.7.0 update to the [Temporal Server](/clusters#temporal-server).
```

Comment:
`ES` is sort of jargon to me. It is ok to use in code but not in documentation. I would use "Elasticsearch" everywhere in our docs.